cas-server-webapp/src/main/webapp/WEB-INF/webflow/login/login-webflow.xml

```diff
@@ -117,6 +118,9 @@
     <view-state id="viewInstitutionLoginForm" view="casInstitutionLoginView" model="credential">
     </view-state>
 
+    <view-state id="viewUnsupportedInstitutionLoginPage" view="casUnsupportedInstitutionLoginView" model="credential">
+    </view-state>
```

Comment:
An inherited issue from my older implementation, I don't think we need the model "credential" at all since neither is an CAS form submission.